From 54da4ba7a8b68c160c45b9745919d0fe2a71feb5 Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Sun, 29 Nov 2020 20:43:25 +0200
Subject: [PATCH] [SW-1833] gaudi: sp sram: set initialized

Set boot status indicating that  SP SRAM is initialized and ready for
use

Change-Id: Ib56431d682a31b7d9d73bde2e33cee8286d920a6
Signed-off-by: Guy Nisan <gnisan@habana.ai>
(cherry picked from commit ebce6ed6dc83b3c72aae9ed206fc90875c05e675)
---

diff --git a/gaudi/common.c b/gaudi/common.c
index e059375..cfbee96 100644
--- a/gaudi/common.c
+++ b/gaudi/common.c
@@ -30,6 +30,7 @@
 #include <preboot/fit.h>
 #include <preboot/thermal.h>
 #include <efuse.h>
+#include <boot_if.h>
 
 #include "common.h"
 
@@ -340,4 +341,6 @@
 	/* SP SRAM access is limited to 32bit */
 	for (i = 0; i < HL_SP_SRAM_SIZE / sizeof(u32); i++)
 		data[i] = 0x5a5a5a5a;
+
+	cpu_boot_dev_sts_clrset(CPU_BOOT_DEV_STS0_SP_SRAM_EN, true);
 }
